examples/cifar10/pt/utils/cifar10_data_splitter.py

```diff
@@ -103,7 +104,10 @@ def split(self, fl_ctx: FLContext):
 
     def load_cifar10_data(self):
         # download data
-        train_dataset = datasets.CIFAR10(root=CIFAR10_ROOT, train=True, download=True)
+        if self.custom_CIFAR10_ROOT:
+            train_dataset = datasets.CIFAR10(root=self.custom_CIFAR10_ROOT, train=True, download=True)
+        else:
+            train_dataset = datasets.CIFAR10(root=CIFAR10_ROOT, train=True, download=True)
```

Comment:
and we can just use same line ``train_dataset = datasets.CIFAR10(root=self.cifar_root, train=True, download=True)``